Firestore/Source/Core/FSTQuery.h

```diff
@@ -55,7 +67,7 @@ typedef NS_ENUM(NSInteger, FSTRelationFilterOperator) {
  * FSTRelationFilter is a document filter constraint on a query with a single relation operator.
  * It is similar to NSComparisonPredicate, except customized for Firestore semantics.
  */
-@interface FSTRelationFilter : NSObject <FSTFilter>
+@interface FSTRelationFilter : FSTFilter
```

Comment:
We can likely convert the factory function on `FSTRelationFilter` to just be an initializer (i.e. rename to `initWith...` and make non-static). Callers then call `[[FSTRelationFilter alloc] initWithField:...]

Reply:
Done.